Get rid of aclocal warning:
/usr/share/aclocal/audiofile.m4:12: warning: underquoted definition of AM_PATH_AUDIOFILE
--- audiofile-0.2.6/audiofile.m4
+++ audiofile-0.2.6/audiofile.m4
@@ -11,3 +11,3 @@
 dnl
-AC_DEFUN(AM_PATH_AUDIOFILE,
+AC_DEFUN([AM_PATH_AUDIOFILE],
 [dnl 
